utility/config.py

```diff
@@ -30,3 +30,14 @@
     "RFAI": "",
     "STAKING": ""
 }
+
+CRYPTO_FIAT_CONVERSION = {
+    'EXCHANGE': 'COINMARKETCAP',
```

Comment:
We should have empty or dummy values here? The actual config needs to be updated in s3 bucket

Reply:
Moved to S3 bucket